recipe/meta.yaml

```diff
@@ -407,6 +407,9 @@ outputs:
         - /usr/lib64/libstdc++.so.6     # [linux]
         # hooray, windows
         - "C:\\Windows\\System32\\WINTRUST.dll"
+    requirements:
+      run:
+        - mpi 1.0 impi
```

Comment:
Do we need this? 

Because this  feedstock also deploy mkl which have blas mutex on top of it and it wasn't defined as run requirements for overall repack package.